releasenotes/notes/add-set-frequency-instruction-18a43ad97a5c6f7f.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Added a new feature. The ``SetFrequency`` instruction allows users to change the
```

Nit: we could make this a link to the SetFrequency class docs with something like:

```suggestion
    Added a new feature. The :py:class:`~qiskit.pulse.instructions.SetFrequency` instruction allows users to change the
```